ruby/ql/test/library-tests/dataflow/api-graphs/callbacks.rb

```diff
@@ -0,0 +1,39 @@
+Something.foo.withCallback do |a, b| #$ use=getMember("Something").getMethod("foo").getReturn().getMethod("withCallback").getReturn()
+    a.something #$ use=getMember("Something").getMethod("foo").getReturn().getMethod("withCallback").getBlock().getParameter(0).getMethod("something").getReturn()
+    b.somethingElse #$ use=getMember("Something").getMethod("foo").getReturn().getMethod("withCallback").getBlock().getParameter(1).getMethod("somethingElse").getReturn()
+end
```

Comment:
Perhaps we should update the test to also cover `def`s. So we would have `def=getMember("Something").getMethod("foo").getReturn().getMethod("withCallback").getBlock()` here (right?)

Reply:
Added `def` tags as optional. It seemed a bit much to require them everywhere.